### PR TITLE
fix: make filed-dropper clickable in firefox

### PR DESCRIPTION
**Pull request by hofmann-felix**
### What was the problem?
* The file dropper wasn't clickable in Firefox (only drop worked)

### How it is solved?
* By adding a label inside the container the file dropper is also clickable in Firefox

Preview URL: https://satellytescommain21751-fixfiledropper.gatsbyjs.io/career/senior-ux-ui-designer/

### CODE REVIEW

File: src/components/forms/file-dropper/file-list-item.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ export const FileListItem = ({
             ))}
           </FileCategoryDropdown>
         )}
-        <RemoveButton onClick={() => onRemove(index)}>
+        <RemoveButton type="button" onClick={() => onRemove(index)}>
```
Comment (hofmann-felix) [inline src/components/forms/file-dropper/file-list-item.tsx:107]: I noticed that when you click remove, the form is sometimes submitted by mistake. This shouldn't happen anymore.

Comment (georgiee) [inline src/components/forms/file-dropper/file-list-item.tsx:107]: But is the type button not implicitly given by the underlying `button`. Why does this change anything?

File: src/components/forms/file-dropper/file-dropper.tsx [hunk 2]
```diff
@@ -197,7 +208,9 @@ export const FileDropper = ({
         hasFiles={hasFiles}
         hasErrors={errors?.documents}
       >
-        <input {...getInputProps({ ...register(name) })} />
+        <StyledLabel>
+          <input {...getInputProps({ ...register(name) })} />
+        </StyledLabel>
```
Comment (georgiee) [inline src/components/forms/file-dropper/file-dropper.tsx:213]: Do you know why that changes the behavior? I mean I understand the thing behind label + input but why does it not work without? How did you came to this fix?

**Review by georgiee — APPROVED** [review]
Thanks it works 🙌
Can you maybe try to answer my questions?

Technically good and approved!

File: src/components/forms/file-dropper/file-list-item.tsx (hunk 1 above)
Comment (hofmann-felix) [reply src/components/forms/file-dropper/file-list-item.tsx:107]: Before the change:
Bildschirm­foto 2023-03-16 um 15 22 36


After the change:
Bildschirm­foto 2023-03-16 um 15 22 57


Apparently `type="button"` is not set implicitly. Currently, if you upload two files on a version without the fix (e.g. https://satellytescommain21751-choreremovestorybook.gatsbyjs.io/career/senior-ux-ui-designer/) and remove the first one with the remove button, the form is submitted. With the fix this is no longer the case.

File: src/components/forms/file-dropper/file-dropper.tsx (hunk 2 above)
Comment (hofmann-felix) [reply src/components/forms/file-dropper/file-dropper.tsx:213]: I found out that there can be problems with file inputs in combination with `diaplay: none`. The solution with the label resulted from trial and error

**Pull request merged**